setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='pynanoleaf',
-    version='0.0.7',
```

Comment:
No need to up the version number here, I'll do that after this PR! Please leave that out.

Reply:
ok